src/Content/Backend/Solution/Monaco.Template.Backend.Api/Endpoints/Products.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Asp.Versioning.Builder;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+#if (!disableAuth)
+using Monaco.Template.Backend.Api.Auth;
+#endif
+using Monaco.Template.Backend.Api.DTOs;
+using Monaco.Template.Backend.Api.DTOs.Extensions;
+using Monaco.Template.Backend.Application.DTOs;
+using Monaco.Template.Backend.Application.Features.Product;
+using Monaco.Template.Backend.Common.Api.Application;
+using Monaco.Template.Backend.Common.Api.MinimalApi;
+using Monaco.Template.Backend.Common.Domain.Model;
+
+namespace Monaco.Template.Backend.Api.Endpoints;
+
+public static class Products
+{
+	public static IEndpointRouteBuilder AddProducts(this IEndpointRouteBuilder builder, ApiVersionSet versionSet)
+	{
+		var products = builder.CreateApiGroupBuilder(versionSet, "Products");
+
+		products.MapGet("",
+						Task<Results<Ok<Page<ProductDto>>, NotFound>> ([FromServices] ISender sender,
+																	   HttpRequest request) =>
+							sender.ExecuteQueryAsync(new GetProductPage.Query(request.Query)),
+						"GetProducts",
+#if (disableAuth)
+						"Gets a page of products");
+#else
+						"Gets a page of products")
+				.AllowAnonymous();
+#endif
+
+		products.MapGet("/{id:guid}",
```

Comment:
I don't think it matters much but here we have `/{id:guid}` and elsewhere we have just `{id:guid}`. They both end up the same but may be clearer to only use one format?

Reply:
Oh, right... I guess it must have been a wild copy/paste 😅 
Fixed.